Fig6_PyGMT_geopandas.py

```diff
@@ -0,0 +1,50 @@
+import geopandas as gpd
+import pygmt
+
+provider = "https://naciscdn.org/naturalearth"
+states = gpd.read_file(f"{provider}/50m/cultural/ne_50m_admin_1_states_provinces.zip")
+rivers = gpd.read_file(f"{provider}/50m/physical/ne_50m_rivers_lake_centerlines.zip")
+cities = gpd.read_file(f"{provider}/110m/cultural/ne_110m_populated_places_simple.zip")
+
+states = states[states["admin"] == "United States of America"].copy()
+states["area_sqkm"] = states.geometry.to_crs(epsg=6933).area / 10 ** 9
+rivers = rivers[rivers.intersects(states.union_all())].copy()
+cities = cities[cities["adm0name"] == "United States of America"].copy()
+
+fig = pygmt.Figure()
+fig.basemap(projection="L-96/35/33/41/12c", region=[-126, -66, 25, 49], frame="+n")
+
+pygmt.makecpt(cmap="bilbao", series=[0, states["area_sqkm"].max()])

```

Comment:
Maybe use a different CPT or a different range, to avoid polygons that are too light or too dark?